### DISCUSSION TITLE
Improvement suggestion about weirdly formatted warning about "unsaved changes"

### DISCUSSION THREAD

**Discussion by ovizii** [mealie-recipes/mealie#3748] [Ideas]
Please take this with a pinch of salt but usually, whenever I encounter an app or a website where I have unsaved changes, the questions if worded so that

OK means proceed without saving
cancel means, stop whatever I was just doing and let me go back.

mealie seems to be doing it differently, meaning both OK and cancel will proceed but one of them without saving. 

Just throwing this out there.

> You have unsaved changes. Do you want to save before leaving? Okay to save, Cancel to discard changes.

![image](https://github.com/mealie-recipes/mealie/assets/837902/e5fa3927-bbcc-440a-961d-db452221b030)

**Comment by ChipSkylark37** [comment]
I thought I was going crazy because I clicked cancel and it exits the page instead of cancelling the page change. Only when I slowed down to read it did I realize it was reversed as you indicated. 

Please either slow an option to "reverse" it to what's normally expected, or change the default so that Cancel keeps you on the page without losing changes.
